pandas/core/indexes/datetimes.py

```diff
@@ -1964,18 +1966,32 @@ def tz_localize(self, tz, ambiguous='raise', errors='raise'):
 
             .. versionadded:: 0.19.0
 
-        infer_dst : boolean, default False
-            .. deprecated:: 0.15.0
-               Attempt to infer fall dst-transition hours based on order
 
         Returns
         -------
         localized : DatetimeIndex
 
+        Examples
+        --------
+        In the example below, We put the date range from 01 March 2018
+        to 08 March 2018 & convert this to US/Eastern Time zone
+
+        >>> df = pd.date_range('2018-03-01', '2018-03-05')
+        >>> df
+        DatetimeIndex(['2018-03-01', '2018-03-02', '2018-03-03', '2018-03-04',
+               '2018-03-05'],
+              dtype='datetime64[ns]', freq='D')
```

Comment:
Can you clean up the alignment a bit here? If I run this in a console, I get this:

```
>>> df
DatetimeIndex(['2018-03-01', '2018-03-02', '2018-03-03', '2018-03-04',
               '2018-03-05'],
              dtype='datetime64[ns]', freq='D')
```

Ideally it should look the same (notice the different vertical alignment with your code)